lib/install/config/webpacker.yml

```diff
@@ -0,0 +1,37 @@
+# Note: You must restart bin/webpack-dev-server for changes to take effect
+
+default: &default
+  dev_server_host: 0.0.0.0
+  dev_server_port: 8080
+  dev_server_https: false
```

Comment:
I've never used cloud9, but I gather that the issue in #176 in wasn't about using a different environment, just configuring the host and port. It looks like apps on cloud9 run in development mode: https://community.c9.io/t/running-a-rails-app/1615